scripts/mcstatus.coffee

```diff
@@ -0,0 +1,48 @@
+# Description:
+#   Report the status of a Minecraft server
+#   Uses http://api.syfaro.net/
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   server_url and server_port, below
+#
+# Commands:
+#   mcstatus - Report the status of the configured minecraft server.
+#
+# Author:
+#   annabunches
+
+default_url = 'mc.wtf.cat'
```

Comment:
Absolutely. Done. And added the ability to specify the host and port directly, if you want to check some arbitrary server. (Is updating my branch sufficient to update the PR?)